AWSIoT/AWSIoTDataManager.h

```diff
@@ -416,6 +416,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)enableMetricsCollection:(BOOL)enabled;
 
+/**
+ Set user-specified dictionary of the additional values to be passed as components of
```

Comment:
Include sample code in the documentation similar to how it is done for other API in this class.

Reply:
Added